web/src/app/pages/edit-survey/edit-survey.component.html

```diff
@@ -130,7 +130,12 @@ <h3 matSubheader class="job-label">Jobs</h3>
         <h1 class="edit-survey-title" *ngIf="sectionTitle !== ''">
           {{ sectionTitle }}
         </h1>
+
         <router-outlet></router-outlet>
+
+        <div class="button-section" *ngIf="router.url.endsWith('/share')">
```

Comment:
The individual components shouldn't need to know about the format of the URL, that's the job of the NavigationService. Let's ask that service instead?